The job **`armhf-gnu`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/13988820147/job/39167959499) [(plain)](https://github.com/rust-lang-ci/rust/commit/a297af71925eac2cf4e6b74fa5535fa6adfae2f1/checks/39167959499/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
file:.git/config remote.origin.url=https://github.com/rust-lang-ci/rust
file:.git/config remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
file:.git/config gc.auto=0
file:.git/config http.https://github.com/.extraheader=AUTHORIZATION: basic ***
file:.git/config branch.try.remote=origin
file:.git/config branch.try.merge=refs/heads/try
file:.git/config remote.upstream.url=https://github.com/rust-lang/rust
file:.git/config remote.upstream.fetch=+refs/heads/*:refs/remotes/upstream/*
file:.git/config submodule.library/backtrace.active=true
file:.git/config submodule.library/backtrace.url=https://github.com/rust-lang/backtrace-rs.git
file:.git/config submodule.library/stdarch.active=true
---
---- [run-make] tests/run-make/doctests-keep-binaries stdout ----

error: rmake recipe failed to complete
status: exit status: 1
command: cd "/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/doctests-keep-binaries/rmake_out" && env -u RUSTFLAGS AR="arm-linux-gnueabihf-ar" BUILD_ROOT="/checkout/obj/build/x86_64-unknown-linux-gnu" CARGO="/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-tools-bin/cargo" CC="arm-linux-gnueabihf-gcc" CC_DEFAULT_FLAGS="-ffunction-sections -fdata-sections -fPIC -march=armv6 -marm -mfpu=vfp" CXX="arm-linux-gnueabihf-g++" CXX_DEFAULT_FLAGS="-ffunction-sections -fdata-sections -fPIC -march=armv6 -marm -mfpu=vfp" HOST_RUSTC_DYLIB_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib" LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib/rustlib/x86_64-unknown-linux-gnu/lib" LD_LIB_PATH_ENVVAR="LD_LIBRARY_PATH" LLVM_BIN_DIR="/checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm/bin" LLVM_COMPONENTS="aarch64 aarch64asmparser aarch64codegen aarch64desc aarch64disassembler aarch64info aarch64utils aggressiveinstcombine all all-targets amdgpu amdgpuasmparser amdgpucodegen amdgpudesc amdgpudisassembler amdgpuinfo amdgputargetmca amdgpuutils analysis arm armasmparser armcodegen armdesc armdisassembler arminfo armutils asmparser asmprinter avr avrasmparser avrcodegen avrdesc avrdisassembler avrinfo binaryformat bitreader bitstreamreader bitwriter bpf bpfasmparser bpfcodegen bpfdesc bpfdisassembler bpfinfo cfguard cgdata codegen codegentypes core coroutines coverage csky cskyasmparser cskycodegen cskydesc cskydisassembler cskyinfo debuginfobtf debuginfocodeview debuginfodwarf debuginfogsym debuginfologicalview debuginfomsf debuginfopdb demangle dlltooldriver dwarflinker dwarflinkerclassic dwarflinkerparallel dwp engine executionengine extensions filecheck frontendatomic frontenddriver frontendhlsl frontendoffloading frontendopenacc frontendopenmp fuzzercli fuzzmutate globalisel hexagon hexagonasmparser hexagoncodegen hexagondesc hexagondisassembler hexagoninfo hipstdpar instcombine instrumentation interfacestub interpreter ipo irprinter irreader jitlink libdriver lineeditor linker loongarch loongarchasmparser loongarchcodegen loongarchdesc loongarchdisassembler loongarchinfo lto m68k m68kasmparser m68kcodegen m68kdesc m68kdisassembler m68kinfo mc mca mcdisassembler mcjit mcparser mips mipsasmparser mipscodegen mipsdesc mipsdisassembler mipsinfo mirparser msp430 msp430asmparser msp430codegen msp430desc msp430disassembler msp430info native nativecodegen nvptx nvptxcodegen nvptxdesc nvptxinfo objcarcopts objcopy object objectyaml option orcdebugging orcjit orcshared orctargetprocess passes powerpc powerpcasmparser powerpccodegen powerpcdesc powerpcdisassembler powerpcinfo profiledata remarks riscv riscvasmparser riscvcodegen riscvdesc riscvdisassembler riscvinfo riscvtargetmca runtimedyld sandboxir scalaropts selectiondag sparc sparcasmparser sparccodegen sparcdesc sparcdisassembler sparcinfo support symbolize systemz systemzasmparser systemzcodegen systemzdesc systemzdisassembler systemzinfo tablegen target targetparser telemetry textapi textapibinaryreader transformutils vectorize webassembly webassemblyasmparser webassemblycodegen webassemblydesc webassemblydisassembler webassemblyinfo webassemblyutils windowsdriver windowsmanifest x86 x86asmparser x86codegen x86desc x86disassembler x86info x86targetmca xray xtensa xtensaasmparser xtensacodegen xtensadesc xtensadisassembler xtensainfo" LLVM_FILECHECK="/checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm/bin/FileCheck" PYTHON="/usr/bin/python3" REMOTE_TEST_CLIENT="/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-tools-bin/remote-test-client" RUSTC="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc" RUSTC_LINKER="arm-linux-gnueabihf-gcc" RUSTDOC="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustdoc" SOURCE_ROOT="/checkout" TARGET="arm-unknown-linux-gnueabihf" TARGET_EXE_DYLIB_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib/rustlib/arm-unknown-linux-gnueabihf/lib" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/doctests-keep-binaries/rmake"
stdout: none
--- stderr -------------------------------
command failed at line 32
Command { cmd: LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/doctests-keep-binaries/rmake_out:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib/rustlib/x86_64-unknown-linux-gnu/lib" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustdoc" "--target=arm-unknown-linux-gnueabihf" "t.rs" "-Zunstable-options" "--test" "--persist-doctests" "doctests" "--extern" "t=libt.rlib", stdin_buf: None, stdin: None, stdout: None, stderr: None, drop_bomb: DropBomb { command: "/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustdoc", defused: true, armed_location: Location { file: "/checkout/tests/run-make/doctests-keep-binaries/rmake.rs", line: 24, col: 9 } }, already_executed: true }
output status: `exit status: 101`
=== STDOUT ===

running 2 tests
test t.rs - florp (line 8) ... FAILED
test t.rs - foople (line 2) ... FAILED

failures:

---- t.rs - florp (line 8) stdout ----
error: linking with `cc` failed: exit status: 1
  |
  = note:  "cc" "/tmp/rustcNH4Kr2/symbols.o" "<2 object files omitted>" "-Wl,--as-needed" "-Wl,-Bstatic" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/doctests-keep-binaries/rmake_out/{libt.rlib}.rlib" "<sysroot>/lib/rustlib/arm-unknown-linux-gnueabihf/lib/{libstd-*,libpanic_unwind-*,libobject-*,libmemchr-*,libaddr2line-*,libgimli-*,librustc_demangle-*,libstd_detect-*,libhashbrown-*,librustc_std_workspace_alloc-*,libminiz_oxide-*,libadler2-*,libunwind-*,libcfg_if-*,liblibc-*,liballoc-*,librustc_std_workspace_core-*,libcore-*,libcompiler_builtins-*}.rlib" "-Wl,-Bdynamic" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-L" "/tmp/rustcNH4Kr2/raw-dylibs" "-Wl,--eh-frame-hdr" "-Wl,-z,noexecstack" "-L" "<sysroot>/lib/rustlib/arm-unknown-linux-gnueabihf/lib" "-o" "doctests/t_rs_8_0/rust_out" "-Wl,--gc-sections" "-pie" "-Wl,-z,relro,-z,now" "-nodefaultlibs"
  = note: some arguments are omitted. use `--verbose` to show all linker arguments
  = note: /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_8_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: error adding symbols: file in wrong format
          collect2: error: ld returned 1 exit status
          

error: aborting due to 1 previous error

Couldn't compile the test.
---- t.rs - foople (line 2) stdout ----
error: linking with `cc` failed: exit status: 1
  |
  = note:  "cc" "/tmp/rustcxtBiVc/symbols.o" "<2 object files omitted>" "-Wl,--as-needed" "-Wl,-Bstatic" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/doctests-keep-binaries/rmake_out/{libt.rlib}.rlib" "<sysroot>/lib/rustlib/arm-unknown-linux-gnueabihf/lib/{libstd-*,libpanic_unwind-*,libobject-*,libmemchr-*,libaddr2line-*,libgimli-*,librustc_demangle-*,libstd_detect-*,libhashbrown-*,librustc_std_workspace_alloc-*,libminiz_oxide-*,libadler2-*,libunwind-*,libcfg_if-*,liblibc-*,liballoc-*,librustc_std_workspace_core-*,libcore-*,libcompiler_builtins-*}.rlib" "-Wl,-Bdynamic" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-L" "/tmp/rustcxtBiVc/raw-dylibs" "-Wl,--eh-frame-hdr" "-Wl,-z,noexecstack" "-L" "<sysroot>/lib/rustlib/arm-unknown-linux-gnueabihf/lib" "-o" "doctests/t_rs_2_0/rust_out" "-Wl,--gc-sections" "-pie" "-Wl,-z,relro,-z,now" "-nodefaultlibs"
  = note: some arguments are omitted. use `--verbose` to show all linker arguments
  = note: /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: relocations in generic ELF (EM: 40)
          /usr/bin/ld: doctests/t_rs_2_0/rust_out.rust_out.116cee38b8060d94-cgu.0.rcgu.o: error adding symbols: file in wrong format
          collect2: error: ld returned 1 exit status
          

error: aborting due to 1 previous error

Couldn't compile the test.

failures:
    t.rs - florp (line 8)
    t.rs - foople (line 2)

test result: FAILED. 0 passed; 2 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.15s



---
---- [run-make] tests/run-make/target-cpu-native stdout ----

error: rmake recipe failed to complete
status: exit status: 1
command: cd "/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/target-cpu-native/rmake_out" && env -u RUSTFLAGS AR="arm-linux-gnueabihf-ar" BUILD_ROOT="/checkout/obj/build/x86_64-unknown-linux-gnu" CARGO="/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-tools-bin/cargo" CC="arm-linux-gnueabihf-gcc" CC_DEFAULT_FLAGS="-ffunction-sections -fdata-sections -fPIC -march=armv6 -marm -mfpu=vfp" CXX="arm-linux-gnueabihf-g++" CXX_DEFAULT_FLAGS="-ffunction-sections -fdata-sections -fPIC -march=armv6 -marm -mfpu=vfp" HOST_RUSTC_DYLIB_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib" LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib/rustlib/x86_64-unknown-linux-gnu/lib" LD_LIB_PATH_ENVVAR="LD_LIBRARY_PATH" LLVM_BIN_DIR="/checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm/bin" LLVM_COMPONENTS="aarch64 aarch64asmparser aarch64codegen aarch64desc aarch64disassembler aarch64info aarch64utils aggressiveinstcombine all all-targets amdgpu amdgpuasmparser amdgpucodegen amdgpudesc amdgpudisassembler amdgpuinfo amdgputargetmca amdgpuutils analysis arm armasmparser armcodegen armdesc armdisassembler arminfo armutils asmparser asmprinter avr avrasmparser avrcodegen avrdesc avrdisassembler avrinfo binaryformat bitreader bitstreamreader bitwriter bpf bpfasmparser bpfcodegen bpfdesc bpfdisassembler bpfinfo cfguard cgdata codegen codegentypes core coroutines coverage csky cskyasmparser cskycodegen cskydesc cskydisassembler cskyinfo debuginfobtf debuginfocodeview debuginfodwarf debuginfogsym debuginfologicalview debuginfomsf debuginfopdb demangle dlltooldriver dwarflinker dwarflinkerclassic dwarflinkerparallel dwp engine executionengine extensions filecheck frontendatomic frontenddriver frontendhlsl frontendoffloading frontendopenacc frontendopenmp fuzzercli fuzzmutate globalisel hexagon hexagonasmparser hexagoncodegen hexagondesc hexagondisassembler hexagoninfo hipstdpar instcombine instrumentation interfacestub interpreter ipo irprinter irreader jitlink libdriver lineeditor linker loongarch loongarchasmparser loongarchcodegen loongarchdesc loongarchdisassembler loongarchinfo lto m68k m68kasmparser m68kcodegen m68kdesc m68kdisassembler m68kinfo mc mca mcdisassembler mcjit mcparser mips mipsasmparser mipscodegen mipsdesc mipsdisassembler mipsinfo mirparser msp430 msp430asmparser msp430codegen msp430desc msp430disassembler msp430info native nativecodegen nvptx nvptxcodegen nvptxdesc nvptxinfo objcarcopts objcopy object objectyaml option orcdebugging orcjit orcshared orctargetprocess passes powerpc powerpcasmparser powerpccodegen powerpcdesc powerpcdisassembler powerpcinfo profiledata remarks riscv riscvasmparser riscvcodegen riscvdesc riscvdisassembler riscvinfo riscvtargetmca runtimedyld sandboxir scalaropts selectiondag sparc sparcasmparser sparccodegen sparcdesc sparcdisassembler sparcinfo support symbolize systemz systemzasmparser systemzcodegen systemzdesc systemzdisassembler systemzinfo tablegen target targetparser telemetry textapi textapibinaryreader transformutils vectorize webassembly webassemblyasmparser webassemblycodegen webassemblydesc webassemblydisassembler webassemblyinfo webassemblyutils windowsdriver windowsmanifest x86 x86asmparser x86codegen x86desc x86disassembler x86info x86targetmca xray xtensa xtensaasmparser xtensacodegen xtensadesc xtensadisassembler xtensainfo" LLVM_FILECHECK="/checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm/bin/FileCheck" PYTHON="/usr/bin/python3" REMOTE_TEST_CLIENT="/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-tools-bin/remote-test-client" RUSTC="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc" RUSTC_LINKER="arm-linux-gnueabihf-gcc" RUSTDOC="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustdoc" SOURCE_ROOT="/checkout" TARGET="arm-unknown-linux-gnueabihf" TARGET_EXE_DYLIB_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib/rustlib/arm-unknown-linux-gnueabihf/lib" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/target-cpu-native/rmake"
stdout: none
--- stderr -------------------------------
command failed at line 10
Command { cmd: LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/target-cpu-native/rmake_out:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib/rustlib/x86_64-unknown-linux-gnu/lib" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc" "-L" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/target-cpu-native/rmake_out" "--target=arm-unknown-linux-gnueabihf" "foo.rs" "-Ctarget-cpu=native", stdin_buf: None, stdin: None, stdout: None, stderr: None, drop_bomb: DropBomb { command: "/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc", defused: true, armed_location: Location { file: "/checkout/tests/run-make/target-cpu-native/rmake.rs", line: 10, col: 9 } }, already_executed: true }
output status: `exit status: 1`
=== STDOUT ===



=== STDERR ===
'znver3' is not a recognized processor for this target (ignoring processor)
'+prfchw' is not a recognized feature for this target (ignoring feature)
'-cldemote' is not a recognized feature for this target (ignoring feature)
'+avx' is not a recognized feature for this target (ignoring feature)
'+sahf' is not a recognized feature for this target (ignoring feature)
'+pclmul' is not a recognized feature for this target (ignoring feature)
'-xop' is not a recognized feature for this target (ignoring feature)
'+crc32' is not a recognized feature for this target (ignoring feature)
'-amx-fp8' is not a recognized feature for this target (ignoring feature)
'+xsaves' is not a recognized feature for this target (ignoring feature)
'-avx512fp16' is not a recognized feature for this target (ignoring feature)
'-usermsr' is not a recognized feature for this target (ignoring feature)
'-sm4' is not a recognized feature for this target (ignoring feature)
'-egpr' is not a recognized feature for this target (ignoring feature)
'+sse4.1' is not a recognized feature for this target (ignoring feature)
'-avx512ifma' is not a recognized feature for this target (ignoring feature)
'+xsave' is not a recognized feature for this target (ignoring feature)
'+sse4.2' is not a recognized feature for this target (ignoring feature)
'-tsxldtrk' is not a recognized feature for this target (ignoring feature)
'-sm3' is not a recognized feature for this target (ignoring feature)
'-ptwrite' is not a recognized feature for this target (ignoring feature)
'-widekl' is not a recognized feature for this target (ignoring feature)
'-movrs' is not a recognized feature for this target (ignoring feature)
'+invpcid' is not a recognized feature for this target (ignoring feature)
'+64bit' is not a recognized feature for this target (ignoring feature)
'+xsavec' is not a recognized feature for this target (ignoring feature)
'-avx10.1-512' is not a recognized feature for this target (ignoring feature)
'-avx512vpopcntdq' is not a recognized feature for this target (ignoring feature)
'+cmov' is not a recognized feature for this target (ignoring feature)
'-avx512vp2intersect' is not a recognized feature for this target (ignoring feature)
'-avx512cd' is not a recognized feature for this target (ignoring feature)
'+movbe' is not a recognized feature for this target (ignoring feature)
'-avxvnniint8' is not a recognized feature for this target (ignoring feature)
'-ccmp' is not a recognized feature for this target (ignoring feature)
'-amx-int8' is not a recognized feature for this target (ignoring feature)
'-kl' is not a recognized feature for this target (ignoring feature)
'-avx10.1-256' is not a recognized feature for this target (ignoring feature)
'-sha512' is not a recognized feature for this target (ignoring feature)
'-avxvnni' is not a recognized feature for this target (ignoring feature)
'-rtm' is not a recognized feature for this target (ignoring feature)
'+adx' is not a recognized feature for this target (ignoring feature)
'+avx2' is not a recognized feature for this target (ignoring feature)
'-hreset' is not a recognized feature for this target (ignoring feature)
'-movdiri' is not a recognized feature for this target (ignoring feature)
'-serialize' is not a recognized feature for this target (ignoring feature)
'+vpclmulqdq' is not a recognized feature for this target (ignoring feature)
'-avx512vl' is not a recognized feature for this target (ignoring feature)
'-uintr' is not a recognized feature for this target (ignoring feature)
'-cf' is not a recognized feature for this target (ignoring feature)
'+clflushopt' is not a recognized feature for this target (ignoring feature)
'-raoint' is not a recognized feature for this target (ignoring feature)
'-cmpccxadd' is not a recognized feature for this target (ignoring feature)
'+bmi' is not a recognized feature for this target (ignoring feature)
'-amx-tile' is not a recognized feature for this target (ignoring feature)
'+sse' is not a recognized feature for this target (ignoring feature)
'-avx10.2-256' is not a recognized feature for this target (ignoring feature)
'-gfni' is not a recognized feature for this target (ignoring feature)
'-avxvnniint16' is not a recognized feature for this target (ignoring feature)
'-amx-fp16' is not a recognized feature for this target (ignoring feature)
'-zu' is not a recognized feature for this target (ignoring feature)
'-ndd' is not a recognized feature for this target (ignoring feature)
'+xsaveopt' is not a recognized feature for this target (ignoring feature)
'+rdrnd' is not a recognized feature for this target (ignoring feature)
'-avx512f' is not a recognized feature for this target (ignoring feature)
'-amx-bf16' is not a recognized feature for this target (ignoring feature)
'-avx512bf16' is not a recognized feature for this target (ignoring feature)
'-avx512vnni' is not a recognized feature for this target (ignoring feature)
'-push2pop2' is not a recognized feature for this target (ignoring feature)
'+cx8' is not a recognized feature for this target (ignoring feature)
'-avx512bw' is not a recognized feature for this target (ignoring feature)
'+sse3' is not a recognized feature for this target (ignoring feature)
'-pku' is not a recognized feature for this target (ignoring feature)
'-nf' is not a recognized feature for this target (ignoring feature)
'-amx-tf32' is not a recognized feature for this target (ignoring feature)
'-amx-avx512' is not a recognized feature for this target (ignoring feature)
'+fsgsbase' is not a recognized feature for this target (ignoring feature)
'+clzero' is not a recognized feature for this target (ignoring feature)
'-mwaitx' is not a recognized feature for this target (ignoring feature)
'-lwp' is not a recognized feature for this target (ignoring feature)
'+lzcnt' is not a recognized feature for this target (ignoring feature)
'+sha' is not a recognized feature for this target (ignoring feature)
'-movdir64b' is not a recognized feature for this target (ignoring feature)
'-ppx' is not a recognized feature for this target (ignoring feature)
'-wbnoinvd' is not a recognized feature for this target (ignoring feature)
'-enqcmd' is not a recognized feature for this target (ignoring feature)
'-amx-transpose' is not a recognized feature for this target (ignoring feature)
'-avx10.2-512' is not a recognized feature for this target (ignoring feature)
'-avxneconvert' is not a recognized feature for this target (ignoring feature)
'-tbm' is not a recognized feature for this target (ignoring feature)
'-pconfig' is not a recognized feature for this target (ignoring feature)
'-amx-complex' is not a recognized feature for this target (ignoring feature)
'+ssse3' is not a recognized feature for this target (ignoring feature)
'+cx16' is not a recognized feature for this target (ignoring feature)
'+bmi2' is not a recognized feature for this target (ignoring feature)
'+fma' is not a recognized feature for this target (ignoring feature)
'+popcnt' is not a recognized feature for this target (ignoring feature)
'-avxifma' is not a recognized feature for this target (ignoring feature)
'+f16c' is not a recognized feature for this target (ignoring feature)
'-avx512bitalg' is not a recognized feature for this target (ignoring feature)
'+rdpru' is not a recognized feature for this target (ignoring feature)
'+clwb' is not a recognized feature for this target (ignoring feature)
'+mmx' is not a recognized feature for this target (ignoring feature)
'+sse2' is not a recognized feature for this target (ignoring feature)
'+rdseed' is not a recognized feature for this target (ignoring feature)
'-avx512vbmi2' is not a recognized feature for this target (ignoring feature)
'-prefetchi' is not a recognized feature for this target (ignoring feature)
'-amx-movrs' is not a recognized feature for this target (ignoring feature)
'+rdpid' is not a recognized feature for this target (ignoring feature)
'-fma4' is not a recognized feature for this target (ignoring feature)
'-avx512vbmi' is not a recognized feature for this target (ignoring feature)
'+shstk' is not a recognized feature for this target (ignoring feature)
'+vaes' is not a recognized feature for this target (ignoring feature)
'-waitpkg' is not a recognized feature for this target (ignoring feature)
'-sgx' is not a recognized feature for this target (ignoring feature)
'+fxsr' is not a recognized feature for this target (ignoring feature)
'-avx512dq' is not a recognized feature for this target (ignoring feature)
'+sse4a' is not a recognized feature for this target (ignoring feature)
'znver3' is not a recognized processor for this target (ignoring processor)
'znver3' is not a recognized processor for this target (ignoring processor)
'+prfchw' is not a recognized feature for this target (ignoring feature)
'-cldemote' is not a recognized feature for this target (ignoring feature)
'+avx' is not a recognized feature for this target (ignoring feature)
'+sahf' is not a recognized feature for this target (ignoring feature)
'+pclmul' is not a recognized feature for this target (ignoring feature)
'-xop' is not a recognized feature for this target (ignoring feature)
'+crc32' is not a recognized feature for this target (ignoring feature)
'-amx-fp8' is not a recognized feature for this target (ignoring feature)
'+xsaves' is not a recognized feature for this target (ignoring feature)
'-avx512fp16' is not a recognized feature for this target (ignoring feature)
'-usermsr' is not a recognized feature for this target (ignoring feature)
'-sm4' is not a recognized feature for this target (ignoring feature)
'-egpr' is not a recognized feature for this target (ignoring feature)
'+sse4.1' is not a recognized feature for this target (ignoring feature)
'-avx512ifma' is not a recognized feature for this target (ignoring feature)
'+xsave' is not a recognized feature for this target (ignoring feature)
'+sse4.2' is not a recognized feature for this target (ignoring feature)
'-tsxldtrk' is not a recognized feature for this target (ignoring feature)
'-sm3' is not a recognized feature for this target (ignoring feature)
'-ptwrite' is not a recognized feature for this target (ignoring feature)
'-widekl' is not a recognized feature for this target (ignoring feature)
'-movrs' is not a recognized feature for this target (ignoring feature)
'+invpcid' is not a recognized feature for this target (ignoring feature)
'+64bit' is not a recognized feature for this target (ignoring feature)
'+xsavec' is not a recognized feature for this target (ignoring feature)
'-avx10.1-512' is not a recognized feature for this target (ignoring feature)
'-avx512vpopcntdq' is not a recognized feature for this target (ignoring feature)
'+cmov' is not a recognized feature for this target (ignoring feature)
'-avx512vp2intersect' is not a recognized feature for this target (ignoring feature)
'-avx512cd' is not a recognized feature for this target (ignoring feature)
'+movbe' is not a recognized feature for this target (ignoring feature)
'-avxvnniint8' is not a recognized feature for this target (ignoring feature)
'-ccmp' is not a recognized feature for this target (ignoring feature)
'-amx-int8' is not a recognized feature for this target (ignoring feature)
'-kl' is not a recognized feature for this target (ignoring feature)
'-avx10.1-256' is not a recognized feature for this target (ignoring feature)
'-sha512' is not a recognized feature for this target (ignoring feature)
'-avxvnni' is not a recognized feature for this target (ignoring feature)
'-rtm' is not a recognized feature for this target (ignoring feature)
'+adx' is not a recognized feature for this target (ignoring feature)
'+avx2' is not a recognized feature for this target (ignoring feature)
'-hreset' is not a recognized feature for this target (ignoring feature)
'-movdiri' is not a recognized feature for this target (ignoring feature)
'-serialize' is not a recognized feature for this target (ignoring feature)
'+vpclmulqdq' is not a recognized feature for this target (ignoring feature)
'-avx512vl' is not a recognized feature for this target (ignoring feature)
'-uintr' is not a recognized feature for this target (ignoring feature)
'-cf' is not a recognized feature for this target (ignoring feature)
'+clflushopt' is not a recognized feature for this target (ignoring feature)
'-raoint' is not a recognized feature for this target (ignoring feature)
'-cmpccxadd' is not a recognized feature for this target (ignoring feature)
'+bmi' is not a recognized feature for this target (ignoring feature)
'-amx-tile' is not a recognized feature for this target (ignoring feature)
'+sse' is not a recognized feature for this target (ignoring feature)
'-avx10.2-256' is not a recognized feature for this target (ignoring feature)
'-gfni' is not a recognized feature for this target (ignoring feature)
'-avxvnniint16' is not a recognized feature for this target (ignoring feature)
'-amx-fp16' is not a recognized feature for this target (ignoring feature)
'-zu' is not a recognized feature for this target (ignoring feature)
'-ndd' is not a recognized feature for this target (ignoring feature)
'+xsaveopt' is not a recognized feature for this target (ignoring feature)
'+rdrnd' is not a recognized feature for this target (ignoring feature)
'-avx512f' is not a recognized feature for this target (ignoring feature)
'-amx-bf16' is not a recognized feature for this target (ignoring feature)
'-avx512bf16' is not a recognized feature for this target (ignoring feature)
'-avx512vnni' is not a recognized feature for this target (ignoring feature)
'-push2pop2' is not a recognized feature for this target (ignoring feature)
'+cx8' is not a recognized feature for this target (ignoring feature)
'-avx512bw' is not a recognized feature for this target (ignoring feature)
'+sse3' is not a recognized feature for this target (ignoring feature)
'-pku' is not a recognized feature for this target (ignoring feature)
'-nf' is not a recognized feature for this target (ignoring feature)
'-amx-tf32' is not a recognized feature for this target (ignoring feature)
'-amx-avx512' is not a recognized feature for this target (ignoring feature)
'+fsgsbase' is not a recognized feature for this target (ignoring feature)
'+clzero' is not a recognized feature for this target (ignoring feature)
'-mwaitx' is not a recognized feature for this target (ignoring feature)
'-lwp' is not a recognized feature for this target (ignoring feature)
'+lzcnt' is not a recognized feature for this target (ignoring feature)
'+sha' is not a recognized feature for this target (ignoring feature)
'-movdir64b' is not a recognized feature for this target (ignoring feature)
'-ppx' is not a recognized feature for this target (ignoring feature)
'-wbnoinvd' is not a recognized feature for this target (ignoring feature)
'-enqcmd' is not a recognized feature for this target (ignoring feature)
'-amx-transpose' is not a recognized feature for this target (ignoring feature)
'-avx10.2-512' is not a recognized feature for this target (ignoring feature)
'-avxneconvert' is not a recognized feature for this target (ignoring feature)
'-tbm' is not a recognized feature for this target (ignoring feature)
'-pconfig' is not a recognized feature for this target (ignoring feature)
'-amx-complex' is not a recognized feature for this target (ignoring feature)
'+ssse3' is not a recognized feature for this target (ignoring feature)
'+cx16' is not a recognized feature for this target (ignoring feature)
'+bmi2' is not a recognized feature for this target (ignoring feature)
'+fma' is not a recognized feature for this target (ignoring feature)
'+popcnt' is not a recognized feature for this target (ignoring feature)
'-avxifma' is not a recognized feature for this target (ignoring feature)
'+f16c' is not a recognized feature for this target (ignoring feature)
'-avx512bitalg' is not a recognized feature for this target (ignoring feature)
'+rdpru' is not a recognized feature for this target (ignoring feature)
'+clwb' is not a recognized feature for this target (ignoring feature)
'+mmx' is not a recognized feature for this target (ignoring feature)
'+sse2' is not a recognized feature for this target (ignoring feature)
'+rdseed' is not a recognized feature for this target (ignoring feature)
'-avx512vbmi2' is not a recognized feature for this target (ignoring feature)
'-prefetchi' is not a recognized feature for this target (ignoring feature)
'-amx-movrs' is not a recognized feature for this target (ignoring feature)
'+rdpid' is not a recognized feature for this target (ignoring feature)
'-fma4' is not a recognized feature for this target (ignoring feature)
'-avx512vbmi' is not a recognized feature for this target (ignoring feature)
'+shstk' is not a recognized feature for this target (ignoring feature)
'+vaes' is not a recognized feature for this target (ignoring feature)
'-waitpkg' is not a recognized feature for this target (ignoring feature)
'-sgx' is not a recognized feature for this target (ignoring feature)
'+fxsr' is not a recognized feature for this target (ignoring feature)
'-avx512dq' is not a recognized feature for this target (ignoring feature)
'+sse4a' is not a recognized feature for this target (ignoring feature)
'znver3' is not a recognized processor for this target (ignoring processor)
'znver3' is not a recognized processor for this target (ignoring processor)
'+prfchw' is not a recognized feature for this target (ignoring feature)
'-cldemote' is not a recognized feature for this target (ignoring feature)
'+avx' is not a recognized feature for this target (ignoring feature)
'+sahf' is not a recognized feature for this target (ignoring feature)
'+pclmul' is not a recognized feature for this target (ignoring feature)
'-xop' is not a recognized feature for this target (ignoring feature)
'+crc32' is not a recognized feature for this target (ignoring feature)
'-amx-fp8' is not a recognized feature for this target (ignoring feature)
'+xsaves' is not a recognized feature for this target (ignoring feature)
'-avx512fp16' is not a recognized feature for this target (ignoring feature)
'-usermsr' is not a recognized feature for this target (ignoring feature)
'-sm4' is not a recognized feature for this target (ignoring feature)
'-egpr' is not a recognized feature for this target (ignoring feature)
'+sse4.1' is not a recognized feature for this target (ignoring feature)
'-avx512ifma' is not a recognized feature for this target (ignoring feature)
'+xsave' is not a recognized feature for this target (ignoring feature)
'+sse4.2' is not a recognized feature for this target (ignoring feature)
'-tsxldtrk' is not a recognized feature for this target (ignoring feature)
'-sm3' is not a recognized feature for this target (ignoring feature)
'-ptwrite' is not a recognized feature for this target (ignoring feature)
'-widekl' is not a recognized feature for this target (ignoring feature)
'-movrs' is not a recognized feature for this target (ignoring feature)
'+invpcid' is not a recognized feature for this target (ignoring feature)
'+64bit' is not a recognized feature for this target (ignoring feature)
'+xsavec' is not a recognized feature for this target (ignoring feature)
'-avx10.1-512' is not a recognized feature for this target (ignoring feature)
'-avx512vpopcntdq' is not a recognized feature for this target (ignoring feature)
'+cmov' is not a recognized feature for this target (ignoring feature)
'-avx512vp2intersect' is not a recognized feature for this target (ignoring feature)
'-avx512cd' is not a recognized feature for this target (ignoring feature)
'+movbe' is not a recognized feature for this target (ignoring feature)
'-avxvnniint8' is not a recognized feature for this target (ignoring feature)
'-ccmp' is not a recognized feature for this target (ignoring feature)
'-amx-int8' is not a recognized feature for this target (ignoring feature)
'-kl' is not a recognized feature for this target (ignoring feature)
'-avx10.1-256' is not a recognized feature for this target (ignoring feature)
'-sha512' is not a recognized feature for this target (ignoring feature)
'-avxvnni' is not a recognized feature for this target (ignoring feature)
'-rtm' is not a recognized feature for this target (ignoring feature)
'+adx' is not a recognized feature for this target (ignoring feature)
'+avx2' is not a recognized feature for this target (ignoring feature)
'-hreset' is not a recognized feature for this target (ignoring feature)
'-movdiri' is not a recognized feature for this target (ignoring feature)
'-serialize' is not a recognized feature for this target (ignoring feature)
'+vpclmulqdq' is not a recognized feature for this target (ignoring feature)
'-avx512vl' is not a recognized feature for this target (ignoring feature)
'-uintr' is not a recognized feature for this target (ignoring feature)
'-cf' is not a recognized feature for this target (ignoring feature)
'+clflushopt' is not a recognized feature for this target (ignoring feature)
'-raoint' is not a recognized feature for this target (ignoring feature)
'-cmpccxadd' is not a recognized feature for this target (ignoring feature)
'+bmi' is not a recognized feature for this target (ignoring feature)
'-amx-tile' is not a recognized feature for this target (ignoring feature)
'+sse' is not a recognized feature for this target (ignoring feature)
'-avx10.2-256' is not a recognized feature for this target (ignoring feature)
'-gfni' is not a recognized feature for this target (ignoring feature)
'-avxvnniint16' is not a recognized feature for this target (ignoring feature)
'-amx-fp16' is not a recognized feature for this target (ignoring feature)
'-zu' is not a recognized feature for this target (ignoring feature)
'-ndd' is not a recognized feature for this target (ignoring feature)
'+xsaveopt' is not a recognized feature for this target (ignoring feature)
'+rdrnd' is not a recognized feature for this target (ignoring feature)
'-avx512f' is not a recognized feature for this target (ignoring feature)
'-amx-bf16' is not a recognized feature for this target (ignoring feature)
'-avx512bf16' is not a recognized feature for this target (ignoring feature)
'-avx512vnni' is not a recognized feature for this target (ignoring feature)
'-push2pop2' is not a recognized feature for this target (ignoring feature)
'+cx8' is not a recognized feature for this target (ignoring feature)
'-avx512bw' is not a recognized feature for this target (ignoring feature)
'+sse3' is not a recognized feature for this target (ignoring feature)
'-pku' is not a recognized feature for this target (ignoring feature)
'-nf' is not a recognized feature for this target (ignoring feature)
'-amx-tf32' is not a recognized feature for this target (ignoring feature)
'-amx-avx512' is not a recognized feature for this target (ignoring feature)
'+fsgsbase' is not a recognized feature for this target (ignoring feature)
'+clzero' is not a recognized feature for this target (ignoring feature)
'-mwaitx' is not a recognized feature for this target (ignoring feature)
'-lwp' is not a recognized feature for this target (ignoring feature)
'+lzcnt' is not a recognized feature for this target (ignoring feature)
'+sha' is not a recognized feature for this target (ignoring feature)
'-movdir64b' is not a recognized feature for this target (ignoring feature)
'-ppx' is not a recognized feature for this target (ignoring feature)
'-wbnoinvd' is not a recognized feature for this target (ignoring feature)
'-enqcmd' is not a recognized feature for this target (ignoring feature)
'-amx-transpose' is not a recognized feature for this target (ignoring feature)
'-avx10.2-512' is not a recognized feature for this target (ignoring feature)
'-avxneconvert' is not a recognized feature for this target (ignoring feature)
'-tbm' is not a recognized feature for this target (ignoring feature)
'-pconfig' is not a recognized feature for this target (ignoring feature)
'-amx-complex' is not a recognized feature for this target (ignoring feature)
'+ssse3' is not a recognized feature for this target (ignoring feature)
'+cx16' is not a recognized feature for this target (ignoring feature)
'+bmi2' is not a recognized feature for this target (ignoring feature)
'+fma' is not a recognized feature for this target (ignoring feature)
'+popcnt' is not a recognized feature for this target (ignoring feature)
'-avxifma' is not a recognized feature for this target (ignoring feature)
'+f16c' is not a recognized feature for this target (ignoring feature)
'-avx512bitalg' is not a recognized feature for this target (ignoring feature)
'+rdpru' is not a recognized feature for this target (ignoring feature)
'+clwb' is not a recognized feature for this target (ignoring feature)
'+mmx' is not a recognized feature for this target (ignoring feature)
'+sse2' is not a recognized feature for this target (ignoring feature)
'+rdseed' is not a recognized feature for this target (ignoring feature)
'-avx512vbmi2' is not a recognized feature for this target (ignoring feature)
'-prefetchi' is not a recognized feature for this target (ignoring feature)
'-amx-movrs' is not a recognized feature for this target (ignoring feature)
'+rdpid' is not a recognized feature for this target (ignoring feature)
'-fma4' is not a recognized feature for this target (ignoring feature)
'-avx512vbmi' is not a recognized feature for this target (ignoring feature)
'+shstk' is not a recognized feature for this target (ignoring feature)
'+vaes' is not a recognized feature for this target (ignoring feature)
'-waitpkg' is not a recognized feature for this target (ignoring feature)
'-sgx' is not a recognized feature for this target (ignoring feature)
'+fxsr' is not a recognized feature for this target (ignoring feature)
'-avx512dq' is not a recognized feature for this target (ignoring feature)
'+sse4a' is not a recognized feature for this target (ignoring feature)
'znver3' is not a recognized processor for this target (ignoring processor)
'znver3' is not a recognized processor for this target (ignoring processor)
'+prfchw' is not a recognized feature for this target (ignoring feature)
'-cldemote' is not a recognized feature for this target (ignoring feature)
'+avx' is not a recognized feature for this target (ignoring feature)
'+sahf' is not a recognized feature for this target (ignoring feature)
'+pclmul' is not a recognized feature for this target (ignoring feature)
'-xop' is not a recognized feature for this target (ignoring feature)
'+crc32' is not a recognized feature for this target (ignoring feature)
'-amx-fp8' is not a recognized feature for this target (ignoring feature)
'+xsaves' is not a recognized feature for this target (ignoring feature)
'-avx512fp16' is not a recognized feature for this target (ignoring feature)
'-usermsr' is not a recognized feature for this target (ignoring feature)
'-sm4' is not a recognized feature for this target (ignoring feature)
'-egpr' is not a recognized feature for this target (ignoring feature)
'+sse4.1' is not a recognized feature for this target (ignoring feature)
'-avx512ifma' is not a recognized feature for this target (ignoring feature)
'+xsave' is not a recognized feature for this target (ignoring feature)
'+sse4.2' is not a recognized feature for this target (ignoring feature)
'-tsxldtrk' is not a recognized feature for this target (ignoring feature)
'-sm3' is not a recognized feature for this target (ignoring feature)
'-ptwrite' is not a recognized feature for this target (ignoring feature)
'-widekl' is not a recognized feature for this target (ignoring feature)
'-movrs' is not a recognized feature for this target (ignoring feature)
'+invpcid' is not a recognized feature for this target (ignoring feature)
'+64bit' is not a recognized feature for this target (ignoring feature)
'+xsavec' is not a recognized feature for this target (ignoring feature)
'-avx10.1-512' is not a recognized feature for this target (ignoring feature)
'-avx512vpopcntdq' is not a recognized feature for this target (ignoring feature)
'+cmov' is not a recognized feature for this target (ignoring feature)
'-avx512vp2intersect' is not a recognized feature for this target (ignoring feature)
'-avx512cd' is not a recognized feature for this target (ignoring feature)
'+movbe' is not a recognized feature for this target (ignoring feature)
'-avxvnniint8' is not a recognized feature for this target (ignoring feature)
'-ccmp' is not a recognized feature for this target (ignoring feature)
'-amx-int8' is not a recognized feature for this target (ignoring feature)
'-kl' is not a recognized feature for this target (ignoring feature)
'-avx10.1-256' is not a recognized feature for this target (ignoring feature)
'-sha512' is not a recognized feature for this target (ignoring feature)
'-avxvnni' is not a recognized feature for this target (ignoring feature)
'-rtm' is not a recognized feature for this target (ignoring feature)
'+adx' is not a recognized feature for this target (ignoring feature)
'+avx2' is not a recognized feature for this target (ignoring feature)
'-hreset' is not a recognized feature for this target (ignoring feature)
'-movdiri' is not a recognized feature for this target (ignoring feature)
'-serialize' is not a recognized feature for this target (ignoring feature)
'+vpclmulqdq' is not a recognized feature for this target (ignoring feature)
'-avx512vl' is not a recognized feature for this target (ignoring feature)
'-uintr' is not a recognized feature for this target (ignoring feature)
'-cf' is not a recognized feature for this target (ignoring feature)
'+clflushopt' is not a recognized feature for this target (ignoring feature)
'-raoint' is not a recognized feature for this target (ignoring feature)
'-cmpccxadd' is not a recognized feature for this target (ignoring feature)
'+bmi' is not a recognized feature for this target (ignoring feature)
'-amx-tile' is not a recognized feature for this target (ignoring feature)
'+sse' is not a recognized feature for this target (ignoring feature)
'-avx10.2-256' is not a recognized feature for this target (ignoring feature)
'-gfni' is not a recognized feature for this target (ignoring feature)
'-avxvnniint16' is not a recognized feature for this target (ignoring feature)
'-amx-fp16' is not a recognized feature for this target (ignoring feature)
'-zu' is not a recognized feature for this target (ignoring feature)
'-ndd' is not a recognized feature for this target (ignoring feature)
'+xsaveopt' is not a recognized feature for this target (ignoring feature)
'+rdrnd' is not a recognized feature for this target (ignoring feature)
'-avx512f' is not a recognized feature for this target (ignoring feature)
'-amx-bf16' is not a recognized feature for this target (ignoring feature)
'-avx512bf16' is not a recognized feature for this target (ignoring feature)
'-avx512vnni' is not a recognized feature for this target (ignoring feature)
'-push2pop2' is not a recognized feature for this target (ignoring feature)
'+cx8' is not a recognized feature for this target (ignoring feature)
'-avx512bw' is not a recognized feature for this target (ignoring feature)
'+sse3' is not a recognized feature for this target (ignoring feature)
'-pku' is not a recognized feature for this target (ignoring feature)
'-nf' is not a recognized feature for this target (ignoring feature)
'-amx-tf32' is not a recognized feature for this target (ignoring feature)
'-amx-avx512' is not a recognized feature for this target (ignoring feature)
'+fsgsbase' is not a recognized feature for this target (ignoring feature)
'+clzero' is not a recognized feature for this target (ignoring feature)
'-mwaitx' is not a recognized feature for this target (ignoring feature)
'-lwp' is not a recognized feature for this target (ignoring feature)
'+lzcnt' is not a recognized feature for this target (ignoring feature)
'+sha' is not a recognized feature for this target (ignoring feature)
'-movdir64b' is not a recognized feature for this target (ignoring feature)
'-ppx' is not a recognized feature for this target (ignoring feature)
'-wbnoinvd' is not a recognized feature for this target (ignoring feature)
'-enqcmd' is not a recognized feature for this target (ignoring feature)
'-amx-transpose' is not a recognized feature for this target (ignoring feature)
'-avx10.2-512' is not a recognized feature for this target (ignoring feature)
'-avxneconvert' is not a recognized feature for this target (ignoring feature)
'-tbm' is not a recognized feature for this target (ignoring feature)
'-pconfig' is not a recognized feature for this target (ignoring feature)
'-amx-complex' is not a recognized feature for this target (ignoring feature)
'+ssse3' is not a recognized feature for this target (ignoring feature)
'+cx16' is not a recognized feature for this target (ignoring feature)
'+bmi2' is not a recognized feature for this target (ignoring feature)
'+fma' is not a recognized feature for this target (ignoring feature)
'+popcnt' is not a recognized feature for this target (ignoring feature)
'-avxifma' is not a recognized feature for this target (ignoring feature)
'+f16c' is not a recognized feature for this target (ignoring feature)
'-avx512bitalg' is not a recognized feature for this target (ignoring feature)
'+rdpru' is not a recognized feature for this target (ignoring feature)
'+clwb' is not a recognized feature for this target (ignoring feature)
'+mmx' is not a recognized feature for this target (ignoring feature)
'+sse2' is not a recognized feature for this target (ignoring feature)
'+rdseed' is not a recognized feature for this target (ignoring feature)
'-avx512vbmi2' is not a recognized feature for this target (ignoring feature)
'-prefetchi' is not a recognized feature for this target (ignoring feature)
'-amx-movrs' is not a recognized feature for this target (ignoring feature)
'+rdpid' is not a recognized feature for this target (ignoring feature)
'-fma4' is not a recognized feature for this target (ignoring feature)
'-avx512vbmi' is not a recognized feature for this target (ignoring feature)
'+shstk' is not a recognized feature for this target (ignoring feature)
'+vaes' is not a recognized feature for this target (ignoring feature)
'-waitpkg' is not a recognized feature for this target (ignoring feature)
'-sgx' is not a recognized feature for this target (ignoring feature)
'+fxsr' is not a recognized feature for this target (ignoring feature)
'-avx512dq' is not a recognized feature for this target (ignoring feature)
'+sse4a' is not a recognized feature for this target (ignoring feature)
'znver3' is not a recognized processor for this target (ignoring processor)
'znver3' is not a recognized processor for this target (ignoring processor)
'+prfchw' is not a recognized feature for this target (ignoring feature)
'-cldemote' is not a recognized feature for this target (ignoring feature)
'+avx' is not a recognized feature for this target (ignoring feature)
'+sahf' is not a recognized feature for this target (ignoring feature)
'+pclmul' is not a recognized feature for this target (ignoring feature)
'-xop' is not a recognized feature for this target (ignoring feature)
'+crc32' is not a recognized feature for this target (ignoring feature)
'-amx-fp8' is not a recognized feature for this target (ignoring feature)
'+xsaves' is not a recognized feature for this target (ignoring feature)
'-avx512fp16' is not a recognized feature for this target (ignoring feature)
'-usermsr' is not a recognized feature for this target (ignoring feature)
'-sm4' is not a recognized feature for this target (ignoring feature)
```

</details>

--- a/tests/run-make/doctests-keep-binaries/rmake.rs
+++ b/tests/run-make/doctests-keep-binaries/rmake.rs
@@ -1,3 +1,5 @@
+//@ ignore-cross-compile attempts to run the doctests
+
 // Check that valid binaries are persisted by running them, regardless of whether the
 // --run or --no-run option is used.
 

--- a/tests/run-make/target-cpu-native/rmake.rs
+++ b/tests/run-make/target-cpu-native/rmake.rs
@@ -3,6 +3,8 @@
 // warnings when used, and that binaries produced by it can also be successfully executed.
 // See https://github.com/rust-lang/rust/pull/23238
 
+//@ ignore-cross-compile target-cpu=native doesn't work well when cross compiling
+
 use run_make_support::{run, rustc};
 
 fn main() {

